[Waiting Room] Update to the multiple language support section (#6448)
cloudflare/cloudflare-docs: content/waiting-room/how-to/customize-waiting-room.md

#### Multiple-language support

> Customizable waiting rooms can display text in any language supported by the UTF-8 character set. To display estimated wait time, numeric variables like `waitTimeMinutes` and `waitTimeHours` can be used within your waiting room template, regardless of user language. However, at the time, the following variables are only available in English: `waitTimeFormatted`, `timeUntilEventStartFormatted`, and `timeUntilEventEndFormatted`.

Reviewer: ```suggestion
Customizable waiting rooms can display text in any language supported by the UTF-8 character set. To display estimated wait time, you can use numeric variables like `waitTimeMinutes` and `waitTimeHours` within your waiting room template, regardless of user language. However, at the time, the following variables are only available in English: `waitTimeFormatted`, `timeUntilEventStartFormatted`, and `timeUntilEventEndFormatted`.
```
